deploy/samples/project_with_local_audit_logs_dataproc.yaml

```diff
@@ -0,0 +1,94 @@
+# Copyright 2018 Google LLC
```

Comment:
Maybe combine these two samples, and update the comment at the top of the file to describe what is deployed